index.js

```diff
@@ -851,5 +853,6 @@ module.exports = {
   Writable,
   Readable,
   Duplex,
-  Transform
+  Transform,
+  PassThrough
```

Comment:
Can you add a comment here that this is just for compat reasons?

Reply:
Yup, added this